### DISCUSSION TITLE
How I can set contentType field type: time ?

### DISCUSSION THREAD

**Discussion by AndreyChursin** [bolt/core#3446] [Q&A]
How I can set field in contentType with type: time ?
I know about "type: data" and "modify: time" but I need create some fields to show time interval (and filter by this fields)

**Comment by eduardomart** [comment]
I don't think there's a content type `type: time` anymore in Bolt 5

You can use:
```
   type: date
   mode: datetime
```

This is according to the docs:
https://docs.boltcms.io/5.0/fields/date#options

**Reply by AndreyChursin** [reply]
doesn't really fit, how do you create your own data type then? you need to specify 2 values ​​for hours and minutes...

**Comment by AndreyChursin** [comment]
same problem with type geolocation? Or I can modulate them?
